ToNote/ViewModels/MainViewModel.cs

```diff
@@ -72,5 +72,21 @@ public ICommand EditNoteCommand
                 }));
             }
         }
+
+        private ICommand _OpenAboutPageCommand;
+
+        public ICommand OpenAboutPageCommand
+        {
+            get
+            {
+                return _OpenAboutPageCommand ?? (_OpenAboutPageCommand = new RelayCommand(() =>
+                {
+                    var dialog = new AboutPageViewModel();
+
+                    var result = DialogService.OpenDialog(dialog);
```

Comment:
Just a nitpick, but if you're not doing anything with the returned value of a dialog, no need to store it in a variable.

Reply:
Made some corrections